tests/plugins/test_chronological.py

```diff
@@ -0,0 +1,40 @@
+"""Provides unit tests for the `abis_mapping.plugins.chronological` module"""
+
+
+# Third-party
+import frictionless
+
+# Local
+from abis_mapping import plugins
+
+
+def test_checks_valid_chronological_order() -> None:
+    """Test the ChronologicalOrder checker"""
+    # Construct fake resource
+    resource = frictionless.Resource(
+        source=[
+            # Missing
+            {"name": "A", "start": "2022-09-11T15:15:15Z", "end": None},
+            {"name": "B", "start": None, "end": "2022-09-11T15:15:15Z"},
+            {"name": "C", "start": None, "end": None},
+
+            # Valid
+            {"name": "D", "start": "2022-09-11T15:15:15Z", "end": "2023-09-11T15:15:15Z"},
+
+            # Invalid
+            {"name": "E", "start": "2023-09-11T15:15:15Z", "end": "2022-09-11T15:15:15Z"},
```

Comment:
This is a special case of datetime being equal. Maybe name the test accordingly and add the more common case of date.
```
{"name": "End date before start date is invalid, "start": "2023-09-11", "end": "2022-09-01"},
{"name": "Same Date/Time is invalid", "start": "2023-09-11T15:15:15Z", "end": "2022-09-11T15:15:15Z"}
```

Reply:
same dates are valid. added same date situation to 3rd test case table.